index.html

```diff
@@ -24,47 +24,241 @@
+              sign-ups required.
+            </p>
+
+            <div class="second-links">
+              <ul>
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">javascript</a></li>
+              </ul>
+            </div>
+            <div class="see-project">
+              <a href="#">See Project</a>
+            </div>
+          </div>
+        </div>
+        <div id="section-5" class="contain5">
+          <div class="s5">
+            <div class="about-text">
+              <div class="head-text">
+                <h1>About Myself</h1>
+              </div>
+              <p>
+                Hello I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+              </p>
+            </div>
+            <h2>LET'S CONNECT</h2>
+
+            <div class="base-icon-5">
+              <ul>
+                <li>
+                  <a href=""><img src="images/Vector-1.svg" alt="image" /></a>
+                </li>
+                <li>
+                  <a href=""
+                    ><img src="images/Linkedinicon.svg" alt="image"
+                  /></a>
+                </li>
+                <li>
+                  <a href=""><img src="images/Vector-2.svg" alt="image" /></a>
+                </li>
+                <li>
+                  <a href=""><img src="images/Vector-3.svg" alt="image" /></a>
+                </li>
+                <li>
+                  <a href=""><img src="images/Vector.svg" alt="image" /></a>
+                </li>
+              </ul>
+            </div>
+            <div class="resume">
+              <a href="">Get my resume</a>
+            </div>
+            <div class="langv">
+              <ul>
+                <li><h3>Languages</h3></li>
+                <li><img src="images/icdropdown.png" alt="icon" /></li>
+              </ul>
+            </div>
+            <div class="tech5">
+              <ul>
+                <li class="item1">
+                  <img src="images/icjscript.png" alt="image" />
+                </li>
+                <li class="item2">
+                  <img src="images/ichtml.png" alt="image" />
+                </li>
+                <li class="item3">
+                  <img src="images/iccss.png" alt="image" />
+                </li>
+              </ul>
+            </div>
+            <div class="frame">
+              <ul>
+                <li><h3>Frameworks</h3></li>
+                <li><img src="images/Union.png" alt="icon" /></li>
+              </ul>
+            </div>
+            <div class="skill">
+              <ul>
+                <li><h3>Skills</h3></li>
+                <li><img src="images/Union.png" alt="icon" /></li>
+              </ul>
+            </div>
```

Comment:
- [ ] You're almost there, keep going strong. What the previous reviewer meant for you to implement here is to have the
``` html
 <div class="langv">
 <div class="frame">
 <div class="skill">
```
as li's themselves and then you can wrap the entire thing in a ul. This will improve the semantic structure of your website since this is a list of information about you. You can refer to below to see what I mean
``` html
<ul>
     <li class="langv">
     <li class="frame">
     <li class="skill">
</ul>
```